meetings/2022-03-09.md

```diff
@@ -0,0 +1,134 @@
+* ...: If it's happening out of band via the authz agent, client wouldn't need to push it as a claim
+* Matthieu: This whole thread is quite confusing.
+* Wouter: Yes, I think it's a lot of inter-mixed decisions or a lot of different aspects that influence each other.
+* Elf: We have a use case about permissioning applications. The end user is not the resource owner and wants to access and authorise a client. How currently would you resolve that use case with raw ACP? Say the end user only gets read/write from the ACL but it's client.
+* Matthieu: Resource owner has control over ACL and can do as they please.
+* Matthieu: How can I allow a client to gain access to a resource I have only read-write access to if I don't control the resource?
+* ...: That's typically a delegated access use-case.
+* Elf: yes, more like a typical OAuth End user delegates access to an application.
+* Matthieu: Currently in Solid we are thinking of access to resources depending on 3 independent actors all potentially restricted at the resource level.
+  * 1. agent - who can also restrict which client and issuer can be used to gain specific access modes.
+  * 2. client.
+  * 3. issuer.
+* Elf: agent would be the end-user using the client?
+* Matthieu: there could be cases where there is not per say a client, say when you access a resource programatically or via the browser.
+* Elf: In that case (browser) the Solid storage/RS would act as a OAuth client. You'd still have a client.
+* Matthieu: How does that work with curl?
+* Elf: Then curl is your client.
+* Matthieu: How do you uniquely identify those?
+* Elf: You could use client credentials and register the client with your OP and curl would act as that client.
+* Matthieu: And typically client credentials flow doesn't issue uniquely identifiable client ID.
+* Elf: Yes, but even if it's not globally unique (we might still mandate it to be an IRI), even if not dereferenceable as a client ID. But the client would be user-bound. For Solid-OIDC, the client registration would bound the client credentials to a specific user. So the ID Token would have a specific WebID. We still need to define client credentials flow in Solid-OIDC.
+* Wouter: I pretty much agree that there always would be a client. The bigger question is is there always an agent or clients can act on their own behalf. I know that Pavlik believes there always will be agent/user. Which still doesn't answer delegation issue.
+* Pavlik: To clarify I think we always need agent/user, on whose behalf the application is acting. Mostly for any legal policies being applied to that user, software can't bare legal consequences.
+* Matthieu: I think it would be good to look at delegation issue independently from issue of syncing policies to ACL auxiliary resources.
+* Wouter: I think at one point we said that it will be hard to sync Data Grants with structure of ACL for example. At other point we said we would need UMA server to accept data grants as claims.
```

Comment:
```suggestion
* Wouter: I think that at one point, we said that it will be hard to sync Data Grants with structure of ACL for example. At another point, we said we would need UMA server to accept data grants as claims.
```